velox/functions/sparksql/Register.cpp

```diff
@@ -69,6 +70,12 @@ static void workAroundRegistrationMacro(const std::string& prefix) {
 
 namespace sparksql {
 
+template <typename T>
+inline void registerArrayMinMaxFunctions(const std::string& prefix) {
```

Comment:
Why it is inline?

Reply:
this is copy from presto. There isn't much difference with or without `inline`.